jwst/resample/resample_spec_step.py

```diff
@@ -43,7 +43,7 @@ def process(self, input):
         self.drizpars = kwargs
 
         if isinstance(input_models[0], MultiSlitModel):
-            # result is a MultiProductModel
+            # result is a MultiResampModel
```

Comment:
All of these need to be changed to `MultiSlitModel`. We're not creating a `MultiResampModel` after all.